en/config_mc/pid_tuning_guide_multicopter_basic.md

```diff
@@ -0,0 +1,99 @@
+The goal of tuning is to set the P/I/D values such that the _Response_ curve matches the _Setpoint_ curve as closely as possible (i.e. a fast response without overshoots).
+
+![QGC Rate Controller Tuning UI](../../assets/mc_pid_tuning/qgc_mc_pid_tuning_rate_controller.png)
+
+The controllers are layered, which means a higher-level controller passes its results to a lower-level controller.
+The lowest-level controller is the **rate controller**, followed by the **attitude controller**, and finally the **velocity & position controller**.
+The PID tuning needs to be done in this same order, starting with the rate controller, as it will affect all other controllers.
+
+The testing procedure for each controller (rate, attitude, velocity/posibition) and axis (yaw, roll, pitch) is always the same: create a fast setpoint change by moving the sticks very rapidly and observe the response.
+Then adjust the sliders (as discussed below) to improve the tracking of the response to the setpoint.
+
+:::tip
+- Rate controller tuning is the most important, and if tuned well, the other controllers often need no or only minor adjustments
+- Usually the same tuning gains can be used for roll and pitch.
+- use Acro/Stabilized/Altitude mode to tune the rate controller
+- Use Position mode to tune the velocity and position controllers.
+  Make sure to switch to the simple input mode, for direct control and possibility to generate step inputs.
+:::
+
+## Preconditions
+
+- You have selected the closest matching [default airframe configuration](../config/airframe.md) for your vehicle.
+  This should give you a vehicle that already flies.
+- You should have done an [ESC calibration](../advanced_config/esc_calibration.md).
+- If using PWM output: [PWM_MIN](../advanced_config/parameter_reference.md#PWM_MIN) is set correctly.
+  It needs to be set low, but such that the **motors never stop** when the vehicle is armed.
+
+  This can be tested in [Acro mode](../flight_modes/acro_mc.md) or in [Manual/Stabilized mode](../flight_modes/manual_stabilized_mc.md):
+  - Remove propellers
+  - Arm the vehicle and lower the throttle to the minimum
+  - Tilt the vehicle to all directions, about 60 degrees
+  - Check that no motors turn off
+- Use a high-rate telemetry link such as WiFi if at all possible (a typical low-range telemetry radio is not fast enough for real-time feedback and plots).
+  This is particularly important for the rate controller.
+- Disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) before tuning a vehicle.
+
+## Tuning Procedure
+
+The tuning procedure is:
+
+1. Open _QGroundControl_ **Vehicle Setup > PID Tuning**
+   ![QGC Rate Controller Tuning UI](../../assets/mc_pid_tuning/qgc_mc_pid_tuning_rate_controller.png)
+1. Select the **Rate Controller** tab.
+1. Confirm that the airmode selector is set to **Disabled**
+1. Set the *Thrust curve* value to: 0.3
+   :::note
```

Comment:
>> -ie what is thrust curve? What value should be set?

> I'd point to the advanced page, as it requires using logs if you want to tune it really accurate. But w/o any detailed tuning, 0.3 should generally be a good value (the difficulty is that we can't just change the default, as it would invalidate existing setups)

OK. Done here.

FMI, I looked at the docs and they seem to indicate that the curve reflects non-linearity in thrust to PWM signals. In other words, if you had a linear relationship you would expect the tuning at hover thrust to work at higher thrust - correct?
Does that imply that a UAVCAN esc would not have a thrust curve? I.e. for UAVCAN we would set this to 0?

Reply:
Yes that is correct. For UAVCAN it depends: for RPM-based control, you don't need to tune it, but for power-based control (used in most cases), it's the same as for PWM.

Reply:
Fixed in https://github.com/PX4/PX4-user_guide/pull/1115/files#r595679810